src/app/configuration/configuration.component.html

```diff
@@ -0,0 +1,99 @@
+          <input matInput i18n-placeholder placeholder="Username" formControlName="username" required>
+        </mat-form-field>
+      </mat-grid-list>
+      <mat-grid-list cols="1">
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Password" formControlName="password" type="password" required>
+        </mat-form-field>
+      </mat-grid-list>
+      <mat-grid-list cols="1">
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Confirm Password" type="password" formControlName="confirmPassword" required>
+          <mat-error><planet-form-error-messages [control]="loginForm.controls.confirmPassword"></planet-form-error-messages></mat-error>
+        </mat-form-field>
+      </mat-grid-list>
+      <button mat-raised-button color="primary" i18n>Next</button>
+    </form>
+    <div id="login-status">{{message}}</div>
+  </mat-step>
+
+  <mat-step [stepControl]="configurationFormGroup">
+    <form [formGroup]="configurationFormGroup">
+      <ng-template matStepLabel i18n>Set Up Configuration</ng-template>
+      <mat-form-field>
+        <mat-select i18n-placeholder placeholder="Select Nation/Community" (change)="onChange($event.value)" formControlName="planet_type" required>
+          <mat-option value="nation" i18n>Nation</mat-option>
+          <mat-option value="community" i18n>Community</mat-option>
+        </mat-select>
+      </mat-form-field>
+      <div>
+        <div *ngIf="select === 'community'">
+          <mat-form-field>
+              <input matInput i18n-placeholder placeholder="Local Url" formControlName="local_domain" readonly required>
+          </mat-form-field>
+          <mat-form-field>
+            <mat-select i18n-placeholder placeholder="Nation(s)" formControlName="parent_domain" required>
+              <mat-option *ngFor="let nation of nations" [value]="nation.nationurl">{{nation.name}}</mat-option>
+            </mat-select>
+          </mat-form-field>
+        </div>
+        <div *ngIf="select === 'nation'">
+          <mat-form-field>
+              <input matInput i18n-placeholder placeholder="Local Url" formControlName="local_domain" required>
+          </mat-form-field>
+          <mat-form-field>
+              <input matInput i18n-placeholder placeholder="Center" formControlName="parent_domain" required readonly>
+          </mat-form-field>
+        </div>
+        <mat-form-field>
+            <input matInput i18n-placeholder placeholder="Name" formControlName="name" required>
+        </mat-form-field>
+        <mat-form-field>
+            <input matInput i18n-placeholder placeholder="Code" formControlName="code" required>
+        </mat-form-field>
+        <mat-form-field>
+          <input matInput i18n-placeholder placeholder="Language" formControlName="preferred_lang" required>
+        </mat-form-field>
+      </div>
+      <div>
+        <button mat-raised-button color="primary" matStepperPrevious i18n>Back</button>
```

Comment:
When hitting enter on the keyboard this step goes back rather than forward.  Maybe adding `type="button"` to this will fix that.